include/motor-control/core/linear_motion_system.hpp

```diff
@@ -29,6 +29,7 @@ struct LinearMotionSystemConfig {
     MEConfig mech_config{};
     float steps_per_rev{};
     float microstep{};
+    float encoder_ppr{0.0};
```

Comment:
could we make this `encoder_pulse_per_rev` to make it more in line with the other variable names here?